docs/sources/panels/transformations.md

```diff
@@ -222,6 +223,65 @@ After I apply the transformation, my labels appear in the table as fields.
 
 {{< docs-imagebox img="/img/docs/transformations/labels-to-fields-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
 
+
+### Group By
+
+This transformation first groups the data by a specified field value and then processes calculations on each group. The available calculations are the same as the Reduce transformation.
+
+Here's an example of original data.
+
+| Time                | Server ID   | CPU Temperature | Server Status
+|---------------------|-------------|-----------------|----------
+| 2020-07-07 11:34:20 | server 1    | 80              | Shutdown
+| 2020-07-07 11:34:20 | server 3    | 62              | OK
+| 2020-07-07 10:32:20 | server 2    | 90              | Overload
+| 2020-07-07 10:31:22 | server 3    | 55              | OK
+| 2020-07-07 09:30:57 | server 3    | 62              | Rebooting
+| 2020-07-07 09:30:05 | server 2    | 88              | OK
+| 2020-07-07 09:28:06 | server 1    | 80              | OK
+| 2020-07-07 09:25:05 | server 2    | 88              | OK
+| 2020-07-07 09:23:07 | server 1    | 86              | OK
+
+You can understand this transformation by illustrating its two steps. First you specify a field name to group the data by. This will group all the same values of this field together, as if you sorted them. For instance if we **Group By** _Server ID_ :
+
+| Time                | Server ID   | CPU Temperature | Server Status
+|---------------------|-------------|-----------------|----------
+| 2020-07-07 11:34:20 | **server 1**    | 80              | Shutdown
+| 2020-07-07 09:28:06 | **server 1**    | 80              | OK
+| 2020-07-07 09:23:07 | **server 1**    | 86              | OK
+|
+| 2020-07-07 10:32:20 | server 2    | 90              | Overload
+| 2020-07-07 09:30:05 | server 2    | 88              | OK
+| 2020-07-07 09:25:05 | server 2    | 88              | OK
+|
+| 2020-07-07 11:34:20 | ***server 3***    | 62              | OK
+| 2020-07-07 10:31:22 | ***server 3***    | 55              | OK
+| 2020-07-07 09:30:57 | ***server 3***    | 62              | Rebooting
+
+Then you can apply various calculations on each group. The result is reduced to one line per group.
+
+For instance, **On Field** *CPU Temperature*, we can **Calculate** the average. We will obtain the average of each server's CPU Temperature as a result :
```

Comment:
Why is "On Field" bold? Is this a field name? Per the [Documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md) and [Microsoft style guide](https://docs.microsoft.com/en-us/style-guide/welcome/), field names are bold, not italicized. Do not use bold for emphasis.

Reply:
It's the terminology used in the Transformer Editor UI if you take a look at it. Same with **Calculate** and **Group By**. I wanted to highlight them somehow. I haven't read the Documentation style guide. I'll take a look. *CPU Temperature* is a field name.

I might eventually take more time to do this properly with real screen captures, it would make it much clearer to understand.